### PR TITLE
Update documentation to new structure

### PR DESCRIPTION
**Pull request by RolandHewage**
The connector team have agreed on a common format for documentations
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing
This format should be included in the relevant connectors

Resolves https://github.com/wso2-enterprise/choreo/issues/6682

Update documentation to new structure
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing
Add more listener samples
Remove usage of checkpanic in samples

Update documentation to new structure
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing

## Automation tests
 - Unit tests 
   > Done
 - Integration tests
   > Done

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

JDK 11
Ballerina Beta 2
Ubuntu 20.04

### CODE REVIEW

File: asb-ballerina/Ballerina.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ name = "asb"
 version = "0.1.4-SNAPSHOT"
 license= ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = ["Azure","ServiceBus"]
+keywords = ["Azure","Service Bus","Messaging","Message Brokers"]
```
Comment (SkNuwanTissera) [inline asb-ballerina/Ballerina.toml:7]: Don't we need to start with a simple letter? in each keyword

Comment (RolandHewage) [reply asb-ballerina/Ballerina.toml]: Resolved and amended the changes. Thank you for the suggestion.

File: README.md [hunk 2]
```diff
@@ -1,1554 +1,25 @@
...
-   ![image](docs/images/Azure_Service_Bus_Topic_Subscription.png)
-
-## Key Features of Azure Service Bus
-* Load-balancing work across competing workers
-* Safely routing and transferring data and control across service and application boundaries
-* Coordinating transactional work that requires a high-degree of reliability
-
-## Connector Overview
-The Azure Service Bus Ballerina Connector is used to connect to the Azure Service Bus to send and receive messages. 
+The Azure Service Bus [Ballerina](https://ballerina.io/) Connector is used to connect to the Azure Service Bus to send and receive messages.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply README.md:13]: let's rephrase to : "The Azure Service Bus [Ballerina](https://ballerina.io/) connector allows you to connect to the Azure Service Bus to send and receive messages."

File: asb-ballerina/Module.md [hunk 3]
```diff
@@ -1,473 +1,262 @@
-# Ballerina Azure Service Bus Module
+## Overview
 
-Connects to Microsoft Azure Service Bus using Ballerina.
+Ballerina connector for Azure Service Bus is used to connect with the [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/) via Ballerina language which is a fully managed enterprise message broker with message queues and publish-subscribe topics.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply asb-ballerina/Module.md:3]: I am not very clear on what you mean to say here "  
This module also supports asynchronous message listening capabilities from the azure service bus. Service Bus provides a Microsoft supported [native Java API](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus?view=azure-java-stable) (SDK) and this module makes use of this [public API](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true). This public API uses [SAS authentication](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas) and this module supports SAS authentication." 
Are you saying that we are using the native API or the Public API? if only one, then why is the other one important?

[the remark this answers is not in the document]
Comment (shaniR) [reply asb-ballerina/Module.md:3]: Shall we rephrase it to: Ballerina connector for Azure Service Bus is used to connect with the 
Azure Service Bus connector allows you to connect to an Azure Service Bus via the Ballerina Language. 

The Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics.  It provides the capability to send and receive messages from Service Bus queues, topics, and subscriptions. The Azure Service Bus handles messages that include data representing any kind of information, including structured data encoded with the common formats such as the following ones: JSON, XML, Apache Avro, Plain Text.
  
This module also supports asynchronous message listening capabilities from the azure service bus. Service Bus provides a Microsoft supported [native Java API](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus?view=azure-java-stable) (SDK) and this module makes use of this [public API](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true). As the public API applies [SAS authentication](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas), this module supports SAS authentication as well." depending on the above comment?

File: asb-ballerina/Module.md [hunk 4]
```diff
@@ -1,473 +1,262 @@
...
 
-* Java 11 Installed
-  Java Development Kit (JDK) with version 11 is required.
-
-* Ballerina SLAlpha5 Installed
-  Ballerina Swan Lake Alpha 5 is required.
+* Obtain tokens
 
-* Shared Access Signature (SAS) Authentication Credentials
+    Shared Access Signature (SAS) Authentication Credentials are required to communicate with the Azure Service Bus.
```
Comment (shaniR) [inline asb-ballerina/Module.md:28]: Let's remove unnecessary spaces.

Comment (shaniR) [reply asb-ballerina/Module.md:28]: Are these sequential steps? If so shall we add numbering? if not shall we add bullet points?

File: asb-ballerina/Module.md [hunk 5]
```diff
@@ -1,473 +1,262 @@
...
-### Getting the authorization credentials
+    Obtain the authorization credentials:
+    * For Service Bus Queues
 
-#### For Service Bus Queues
+        1. Make sure you have an Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/en-us/free/) before you begin.
 
-1. Make sure you have an Azure subscription. If you don't have an Azure subscription, you can create a
-   [free account](https://azure.microsoft.com/en-us/free/) before you begin.
+        2. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
```
Comment (shaniR) [inline asb-ballerina/Module.md:37]: Let's remove unnecessary space

Comment (shaniR) [reply asb-ballerina/Module.md:37]: This too is mentioned in the prerequisites.

File: asb-ballerina/Module.md [hunk 6]
```diff
@@ -1,473 +1,262 @@
...
 
-2. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
+        3. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
 
-3. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
+        4. [Create a queue in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal). It is in the format ‘queueName’.
```
Comment (shaniR) [inline asb-ballerina/Module.md:41]: Let's remove unnecessary spaces.
This was also a step in the prerequisites. Do we need to add it here?

File: asb-ballerina/Module.md [hunk 7]
```diff
@@ -1,473 +1,262 @@
...
 
-4. [Create a queue in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal). 
-   It is in the format ‘queueName’.
+    * For Service Bus Topics and Subscriptions
```
Comment (shaniR) [inline asb-ballerina/Module.md:43]: Please review this section with the comments added to the section above.

File: asb-ballerina/Module.md [hunk 8]
```diff
@@ -1,473 +1,262 @@
...
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
 
-### Step 5 : Create a Queue Receiver using the Azure service bus client
-You can now make a receiver connection using the connection configuration. Provide the `queueName` as a parameter. 
-Optionally you can provide the receive mode which is PEEKLOCK by default.
-```ballerina
-    handle queueReceiver = checkpanic asbClient->createQueueReceiver(queueName, asb:RECEIVEANDDELETE);
-```
+    Following is an example on how to send messages to the Azure Service Bus using the connector. You need to create a
```
Comment (shaniR) [inline asb-ballerina/Module.md:78]: Let's remove unnecessary spaces

Comment (shaniR) [reply asb-ballerina/Module.md:78]: Let's remove the new line

File: asb-ballerina/Module.md [hunk 9]
```diff
@@ -1,473 +1,262 @@
...
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
 
-### Step 5 : Create a Queue Receiver using the Azure service bus client
-You can now make a receiver connection using the connection configuration. Provide the `queueName` as a parameter. 
-Optionally you can provide the receive mode which is PEEKLOCK by default.
-```ballerina
-    handle queueReceiver = checkpanic asbClient->createQueueReceiver(queueName, asb:RECEIVEANDDELETE);
-```
+    Following is an example on how to send messages to the Azure Service Bus using the connector. You need to create a 
+    `queueSender` before sending a message.
```
Comment (shaniR) [inline asb-ballerina/Module.md:79]: merge this to the above line

File: asb-ballerina/Module.md [hunk 10]
```diff
@@ -1,473 +1,262 @@
...
-    string stringContent = "This is My Message Body"; 
-    byte[] byteContent = stringContent.toBytes();
-    map<string> properties = {a: "propertyValue1", b: "propertyValue2"};
-    int timeToLive = 60; // In seconds
-    int serverWaitTime = 60; // In seconds
-
-    asb:ApplicationProperties applicationProperties = {
-        properties: {a: "propertyValue1", b: "propertyValue2"}
-    };
+    Send a message to the Azure Service Bus
```
Comment (shaniR) [inline asb-ballerina/Module.md:81]: Do we need this, as this is explained above?

File: asb-ballerina/Module.md [hunk 11]
```diff
@@ -1,473 +1,262 @@
...
-```ballerina
-    checkpanic asbClient->closeReceiver(queueReceiver);
-```
+            check asbClient->send(queueSender, message);
+
+            check asbClient->closeSender(queueSender);
+        }
+    ```
+
+    Following is an example on how to receive messages from the Azure Service Bus using the client connector. You need to create a
```
Comment (shaniR) [inline asb-ballerina/Module.md:108]: remove unnecessary spaces.

File: asb-ballerina/Module.md [hunk 12]
```diff
@@ -1,473 +1,262 @@
...
+            check asbClient->send(queueSender, message);
+
+            check asbClient->closeSender(queueSender);
+        }
+    ```
+
+    Following is an example on how to receive messages from the Azure Service Bus using the client connector. You need to create a 
+    `queueReceiver` before receiving a message. Optionally you can provide the receive mode which is PEEKLOCK by default. You can find more information about the receive modes [here](https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.receivemode?view=azure-java-stable).
+
+    Receive a message from the Azure Service Bus
```
Comment (shaniR) [inline asb-ballerina/Module.md:111]: We can move this before the description and add a bullet point. If so, do the same for the above sending a message too.

File: asb-ballerina/Module.md [hunk 13]
```diff
@@ -1,473 +1,262 @@
...
+                log:printError("No message in the queue.");
+            } else {
+                log:printError("Receiving message via Asb receiver connection failed.");
+            }
+
+            check asbClient->closeReceiver(queueReceiver);
+        }
+    ```
+
+    Following is an example on how to asynchronously listen to messages from the Azure Service Bus using the listener.
```
Comment (shaniR) [inline asb-ballerina/Module.md:133]: Shouldn't we add these as full  fledged samples and not in the module.md? Please follow the same format as other connectors.

File: asb-ballerina/Module.md [hunk 14]
```diff
@@ -1,473 +1,262 @@
...
+    You need to create a new listener instance before listening. Then, you need to create a service object with the service 
+    configuration specified using the `@asb:ServiceConfig` annotation and attach it to the listener. You need to give the 
+    connection string and the entity path of the queue we are to listen messages from. We can optionally provide the 
+    receive mode. Default mode is the PEEKLOCK mode. You can find more information about the receive modes 
+    [here](https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.receivemode?view=azure-java-stable). 
+    Finally, you can provide the service logic to execute when a message is received inside the onMessage remote function.
+
+    Listen to Messages from the Azure Service Bus
+
+    **NOTE:**
```
Comment (shaniR) [inline asb-ballerina/Module.md:143]: Notes should follow the markdown syntax.

File: asb-ballerina/Module.md [hunk 15]
```diff
@@ -1,473 +1,262 @@
...
+                    asb:TEXT => {
+                        string stringMessage = check 'string:fromBytes(<byte[]> received);
+                        log:printInfo("The message received: " + stringMessage);
+                    }
+                }
+            }
+        };
+    ```
+
+    **NOTE:**
```
Comment (shaniR) [inline asb-ballerina/Module.md:185]: same here

File: asb-ballerina/Package.md [hunk 16]
```diff
@@ -1,473 +1,22 @@
...
-entity level, rather than at the namespace level (such as sending a message to a queue). This module supports these 
-basic operations. These APIs use SAS authentication and this module supports SAS authentication.  
+The `ballerinax/asb` is a [Ballerina](https://ballerina.io/) connector for Microsoft Azure Service Bus.
 
-# Compatibility
-|                     |    Version                  |
-|:-------------------:|:---------------------------:|
-| Ballerina Language  | Swan-Lake-Alpha5            |
-| Service Bus API     | v3.5.1                      |
+This package provides the capability to easily access Microsoft Azure Service Bus to send and receive messages. You can perform actions such as send to queue, send to topic, receive from queue, receive from subscription, etc.
```
Comment (shaniR) [inline asb-ballerina/Package.md:7]: You can perform operations to send and receive messages from Service Bus queues, topics, and subscriptions

File: README.md [hunk 17]
```diff
@@ -1,1554 +1,25 @@
...
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-azure-service-bus.svg)](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/commits/master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Connects to Microsoft Azure Service Bus using Ballerina.
-
-# Introduction
-
-## What is Azure Service Bus?
-
 The [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/) is a reliable cloud messaging as
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply README.md]: Resolved and amended the changes. Thank you for the suggestion.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (RolandHewage) [reply README.md]: Resolved and amended the changes. Thank you for the suggestion.

File: README.md [hunk 18]
```diff
@@ -1588,11 +59,11 @@ As an open source project, Ballerina welcomes contributions from the community.
 
 For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
 
-## Code of Conduct
+## Code of conduct
 
 All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply README.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 19]
```diff
@@ -1,473 +1,262 @@
...
-
-## Listener Capabilities
-The `ballerinax/asb` module contains operations related to asynchronous message listening capabilities from the azure 
-service bus. It includes operations to attach service, start listening, detach service, graceful stop listening, and 
-immediate stop listening.
-
-# Prerequisites:
-
-* An Azure account and subscription.
+* Create Azure account and subscription.
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 20]
```diff
@@ -1,473 +1,262 @@
...
   If you don't have an Azure subscription, [sign up for a free Azure account](https://azure.microsoft.com/free/).
 
-* A Service Bus namespace.
+* Create Service Bus namespace.
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 21]
```diff
@@ -1,473 +1,262 @@
...
   If you don't have [a service bus namespace](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal),
   learn how to create your Service Bus namespace.
 
-* A messaging entity, such as a queue, topic or subscription.
+* Create messaging entity, such as a queue, topic or subscription.
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 4 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md:30]: This sentence is indented for a proper view. It's not an unnecessary space.

I have added numbering for the sequential steps and bullet points where it's not. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 22]
```diff
@@ -1,473 +1,262 @@
...
+    Shared Access Signature (SAS) Authentication Credentials are required to communicate with the Azure Service Bus.
     * Connection String
     * Entity Path
 
-## Configuration
-Instantiate the connector by giving authorization credentials that a client application can use to send/receive messages
-to/from the queue/topic/subscription.
-
-### Getting the authorization credentials
+    Obtain the authorization credentials:
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md:34]: This sentence is indented for a proper view. It's not an unnecessary space.

File: asb-ballerina/Module.md [hunk 23]
```diff
@@ -1,473 +1,262 @@
...
+    * For Service Bus Queues
 
-#### For Service Bus Queues
+        1. Make sure you have an Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/en-us/free/) before you begin.
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: This is removed now. Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 5 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 24]
```diff
@@ -1,473 +1,262 @@
...
 
-1. Make sure you have an Azure subscription. If you don't have an Azure subscription, you can create a
-   [free account](https://azure.microsoft.com/en-us/free/) before you begin.
+        2. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
 
-2. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
+        3. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: This sentence is indented for a proper view. It's not an unnecessary space.

File: asb-ballerina/Module.md (hunk 6 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md]: This sentence is indented for a proper view. It's not an unnecessary space.

This is not the same subscription term used above for account subscription. This is a technical term.

File: asb-ballerina/Module.md (hunk 7 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md:43]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 25]
```diff
@@ -1,473 +1,262 @@
...
 
-## Send and Receive Messages from the Azure Service Bus Queue
-
-This is the simplest scenario to send and receive messages from an Azure Service Bus queue. You need to obtain 
-a connection string of the name space and an entity path name of the queue you want to send and receive messages from. 
-
-### Step 1: Import the Azure Service Bus Ballerina Library
-First, import the ballerinax/asb module into the Ballerina project.
+### Step 1: Import connector
+First, import the `ballerinax/asb` module into the Ballerina project.
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 8 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 9 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 10 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md:78]: This is followed in the common format. 
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing

File: asb-ballerina/Module.md (hunk 11 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md:105]: This sentence is indented for a proper view. It's not an unnecessary space.

File: asb-ballerina/Module.md (hunk 12 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md:108]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 13 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md:130]: This is a very important scenario. So it's important to add it here. We have agreed to give two examples if there is a client and a listener. Nothing else are added.

File: asb-ballerina/Module.md (hunk 14 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md (hunk 15 above)
Comment (RolandHewage) [reply asb-ballerina/Module.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Package.md (hunk 16 above)
Comment (RolandHewage) [reply asb-ballerina/Package.md]: Resolved and amended the changes. Thank you for the suggestion.

File: asb-ballerina/Module.md [hunk 26]
```diff
@@ -1,473 +1,262 @@
...
 
-# Module Overview
-Microsoft Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe 
-topics. The primary wire protocol for Service Bus is Advanced Messaging Queueing Protocol (AMQP) 1.0, an open ISO/IEC 
-standard. Service Bus provides a Microsoft supported native API and this module make use of this public API. 
-Service Bus service APIs access the Service Bus service directly, and perform various management operations at the 
-entity level, rather than at the namespace level (such as sending a message to a queue). This module supports these 
-basic operations. These APIs use SAS authentication and this module supports SAS authentication.  
+This module supports [Service Bus SDK](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true) 3.5.1 version. The source code on GitHub is located [here](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/servicebus/microsoft-azure-servicebus). The primary wire protocol for Service Bus is Advanced Messaging Queueing Protocol (AMQP) 1.0, an open ISO/IEC standard.
```
[the remark this answers is not in the document]
Comment (RolandHewage) [reply asb-ballerina/Module.md]: It's important to give these details to the user.

**Review by abeykoon — APPROVED** [review]

File: asb-ballerina/client.bal [hunk 27]
```diff
@@ -16,25 +16,27 @@
 
 import ballerina/jballerina.java as java;
 
-# Represents a client to the Azure service bus message broker.
+# Ballerina Service Bus connector provides the capability to access Azure Service Bus SDK.
+# Service Bus API provides data access to highly reliable queues and publish/subscribe topics of Azure Service Bus with deep feature capabilities.
```
Comment (shaniR) [inline asb-ballerina/client.bal:20]: deep feature capabilities?
